learn/core_concepts/relevancy.md

```diff
@@ -114,35 +124,51 @@ Let's say you have a movie dataset. The documents contain the fields `release_da
 The following example will create a rule that makes older movies more relevant than more recent ones. A movie released in 1999 will appear before a movie released in 2020.
 
 ```
-asc(release_date)
+release_date:asc
 ```
 
 The following example will create a rule that makes movies with a good rank more relevant than movies with a lower rank. Movies with a higher ranking will appear first.
 
 ```
-desc(movie_ranking)
+movie_ranking:desc
 ```
 
 To add a rule to the existing ranking rule, you have to add the rule to the existing ordered rules array using the [settings route](/reference/api/ranking_rules.md#update-ranking-rules),
 
 ```json
 [
+  "words",
   "typo",
-  "attribute",
   "proximity",
-  "words",
+  "attribute",
+  "sort",
   "exactness",
-  "asc(release_date)",
-  "desc(movie_ranking)"
+  "release_date:asc",
+  "movie_ranking:desc"
 ]
 ```
 
+### Sorting and custom ranking rules
+
+MeiliSearch allows users to define [sorting order at query time](/reference/features/sorting.md) by using the [`sort` search parameter](/reference/features/search_parameters.md#sort). There is some overlap between sorting and ranking rules, but the two do have different uses.
```

Comment:
```suggestion
MeiliSearch allows users to define [sorting order at query time](/reference/features/sorting.md) by using the [`sort` search parameter](/reference/features/search_parameters.md#sort). There is some overlap between sorting and custom ranking rules, but the two do have different uses.
```

Reply:
Was it initially written for custom ranking rules or more generally for ranking rules?

Reply:
You're absolutely right that it was written with custom ranking rules in mind. Leaving `custom` out was an oversight, thanks for spotting this!